feature/interface/aggregate/otg_tests/balancing_test/balancing_test.go

```diff
@@ -557,7 +561,7 @@ func (tc *testCase) testFlow(t *testing.T, l3header string) {
 	beforeTrafficCounters := tc.getCounters(t, "before")
 
 	tc.ate.OTG().StartTraffic(t)
-	time.Sleep(15 * time.Second)
+	time.Sleep(20 * time.Second)
```

Comment:
You might want to add a little more time.  The traffic rate you set will take 20 seconds, and you are only sleeping for 20 seconds.  If the OTG takes some variable time which is greater than 0 seconds to ramp up to the Pps specified, it will take a little bit longer than 20 seconds to complete.  So this test could easily be flaky unless you allow a bit more time for the traffic to complete.  

Better than sleep though, can you research the OTG api and see if there is a call you can use to block on the traffic flow completing?  This would guarantee the flow is complete versus guessing on a timeout/sleep interval.